keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -610,17 +592,15 @@ metadata:
   name: my-svc
   namespace: my-ns
 spec:
-  ipFamily: IPv4
   ip: 42.42.42.42
+  type: "SuperclusterIP"
   ports:
   - name: http
     protocol: TCP
     port: 80
+  sessionAffinity: None
   clusters:
```

Comment:
this is no longer under spec?

Reply:
sessionAffinity is still under spec, but clusters should be under status (fixed)

Reply:
not pushed?

Reply:
pushed